src/main.rs

```diff
@@ -1,10 +1,13 @@
+#![type_length_limit = "7912782"]
```

Comment:
This was added due to the compile issue #6.

Reply:
Guess we can remove that setting now that you fixed #6.